xbmc/utils/FileUtils.cpp

```diff
@@ -227,4 +227,5 @@ CDateTime CFileUtils::GetModificationDate(const std::string& strFileNameAndPath,
   {
     CLog::Log(LOGERROR, "%s unable to extract modification date for file (%s)", __FUNCTION__, strFileNameAndPath.c_str());
   }
+  return dateAdded;
```

Comment:
It's a really minor. We return in 224 and now we also return at the end. it would be enough to have the return at end.